### PR TITLE
Resolution for MacOS issue related to freetype

### PR DESCRIPTION
**Pull request by mohanr**
The instructions for a straightforward fix when freetype is installed but not detected by the configure process.

### CODE REVIEW

File: pt/known-issues/known_issues_macos.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
 # MacOS
 
+
```
Comment (neomatrix369) [inline pt/known-issues/known_issues_macos.md]: Thanks for the pull request. 

Looks good but I have few styling and formatting hints:
- have a look the way other Known Issues are written and formatted (they are in a Q & A format)
- the styling across commands and other keywords use the ``` markdown formatting

**Comment by thomasmodeneis** [comment]
I've added an issue to cover the translations for this.
Thanks

**Comment by neomatrix369** [comment]
@mohanr - I'm going to merge this and format it whilst integrating it, thanks for your contribution and looking forward to the next one from you.

**Comment by neomatrix369** [comment]
@mohanr - just out of curiosity, did you intend to add the new issue for MacOS to the Portuguese version? As I have moved the changes to the English version and reformatted it to adjust it to the existing format - I hope you are fine with it.

**Comment by mohanr** [comment]
Mani,
    You are right. It should be the english version. My mistake.

Mohan

**Comment by neomatrix369** [comment]
No worries its rectified now, do you want to scan the book once more to
find any other weak links in it ;)

**Comment by mohanr** [comment]
Yes. I was thinking of this section.
https://github.com/neomatrix369/adoptopenjdk-getting-started-kit/blob/master/en/openjdk-projects/jmh.md

Mohan

**Comment by neomatrix369** [comment]
@mohanr  - Sorry I didn't reply earlier, got busy. Thats a great topic to work on. Would you want to collaborate with @mitemitreski, he has contributed that page and also has an understanding of that subject, together you could create something interesting.

**Comment by mohanr** [comment]
Sure. I can do that.

Mohan

**Comment by mohanr** [comment]
I have asked @mitemitreski <https://github.com/mitemitreski> about this.

Thanks,
Mohan

**Pull request merged**